e2e/vue3-vite-e2e/tests/vue3-vite.spec.ts

```diff
@@ -58,4 +60,118 @@ describe('vue3-vite e2e', () => {
       expect(projectJson.tags).toEqual(['e2etag', 'e2ePackage']);
     });
   });
+
+  describe('--alias', () => {
+    it('should use local path alias by default', async () => {
+      // Create app
+      const app = uniq('vue3-vite');
+      await runNxCommandAsync(`generate nx-vue3-vite:app ${app}`);
+
+      // Read paths
+      const tsConfigJson = readJson(`apps/${app}/tsconfig.json`);
+      const { paths } = tsConfigJson.compilerOptions;
+
+      // Verify `tsConfigJson.compilerOptions.paths`
+      expect(Object.keys(paths)).toHaveLength(3);
+      expect(paths).toMatchObject({
+        '@assets/*': ['./src/assets/*'],
+        '@app/*': ['./src/app/*'],
+        '@public/*': ['./src/public/*'],
+      });
+
+      // Verify `vite.config.ts`
+      const viteConfig = readFile(`apps/${app}/vite.config.ts`);
```

Comment:
Hi @sampullman 

What I'm trying to do here is to verify the object exported by `module.exports = defineConfig({ ... })` in `vite.config.ts`.
I didn't find a good way to read the exported object, so currently I'm using an ugly workaround, which is to check if some specific strings are contained in the whole file (as text). :confused:
But it doesn't feel right, is there a recommended solution?

Reply:
I think this is ok! Maybe it's possible to analyze the typescript with something like [this](https://github.com/dsherret/ts-morph/tree/latest/packages/ts-morph), but for this situation I think checking strings is good enough.